conf_checklist.md

```diff
@@ -0,0 +1,88 @@
+# Organization Checklist
+
+## Pre-Conference Communication
+
+- [ ] The Code of Conduct for the conference is written and easily accessible from the conference website.
+
+<sup>Attendees who had first or second-hand problems during conferences might avoid conferences with no CoC.</sup>
+
+- [ ] The website clearly states its goal to defend inclusivity.
+
+<sup>The absence of statement could be interpreted by minorities as an absence of such goal.</sup>
+
+- [ ] The website clearly states whether it is accessible to reduced-mobility attendees.
+
+<sup>Reduced mobility attendees might not engage with the conference when in doubt.</sup>
+
+- [ ] Contact associations and public figures of minority groups directly to advertise the conference.
+
+<sup>Such groups might have parallel networks than the rest of the community and not know about your event otherwise.</sup>
+
+
+## Venue
+
+- [ ] There are at least 5 seats / 100 person, where attendees wait beetween conferences.
+
+<sup>Attendees may have many reasons to need to seat during breaks.</sup>
+
+- [ ] It is possible to reach the venue through large walkways and protected crosswalks.
+
+<sup>Reduced mobility attendees might not be able to reach the venue otherwise.</sup>
+
+- [ ] It is possible to reach the venue and go to all conference rooms without using stairs.
+
+<sup>Some attendees might not be able to climb stairs.</sup>
+
+- [ ] Doors to all conference room are wide enough for a wheelchair to pass.
+
+<sup>Wheelchair users should be able to attend all conferences.</sup>
+
+- [ ] There are gender neutral bathrooms.
+
+<sup>Trans and gender non-conforming attendees should not have to choose gendered toilets.</sup>
+
+- [ ] There are spots where attendees know they can find staff members during the whole conference.
+
+<sup>If a problem arrises for an attendee, they need to be able to find available staff members without needing to search for them.</sup>
+
+- [ ] The conference program is available on an accessible webpage.
+
+<sup>It will allow blind and vision impaired attendees to organize their planning using voice recognition.</sup>
+
+
+## Lunch and Parties
+
+- [ ] Each lunch and party is accessible without using stairs or narrow doors.
+
+<sup>Reduced mobility attendees should also be able to join the parties.</sup>
+
+- [ ] Each lunch has a "first-class" vegetarian option.
```

Comment:
Worth mentioning vegan and gluten-free options as well.

Reply:
I added a "specific diet" line, because I agree, but it quickly becomes unmanageable.